plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -796,6 +796,8 @@ protected final void setup(HostAndPort metastoreAddress, String databaseName)
                         .hdfsEnvironment(hdfsEnvironment)
                         .build()))
                 .executor(executor)
+                .metadataCacheEnabled(true)
```

Comment:
Why using a different setup than the production default here?
add a code comment